addons/web/static/tests/core/position_hook_tests.js

```diff
@@ -78,27 +72,40 @@ QUnit.module("usePosition Hook", {
         });
         TestComp.popperOptions = { container };
 
+        const sheet = document.createElement("style");
+        sheet.innerHTML = `
```

Comment:
Apparently missed it the previous time around, but this is an unnecessary `innerHTML`: `style` elements don't contain markup, so setting `textContent` shoudl be fine?